### DISCUSSION TITLE
How to create radio button (options) while creating new Collection Type new Field?

### DISCUSSION THREAD

**Discussion by luzan** [strapi/strapi#5900] [Legacy Topics]
I am not able to see any option to add radio buttons, is there any way to add it.

**Comment by sunnysonx** [comment]
Its called Enumeration field.

**Comment by luzan** [comment]
> Its called Enumeration field.

I was only able to use the dropdown list using Enumeration field

This is the model I am using
```
{
  "kind": "collectionType",
  "connection": "default",
  "collectionName": "patient",
  "info": {
    "name": "patient"
  },
  "options": {
    "increments": true,
    "timestamps": true
  },
  "attributes": {
    "ReasonForAdmission": {
      "type": "enumeration",
      "enum": [
        "Preterm",
        "NNS_NeonatalSepsis",
        "BirthAsphyxia",
        "NNJ_NeonatalJaundice",
        "Others"
      ]
    }
  }
}
```

**Reply by derrickmehaffy** [reply]
There isn't currently a field type for radio buttons

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Reply by luzan** [reply]
Sorry and Thanks
